index.js

```diff
@@ -11,6 +11,7 @@ const colors = {
   blue: '#7FDBFF',
   lightGray: '#DDDDDD',
   gray: '#AAAAAA',
+  orange: '#FF851B',
```

Comment:
Yay! Though ideally we'd get like the full warning yellow background color to the messages I think - perhaps in a separate PR :grin:

<img width="1268" alt="screen shot 2016-07-20 at 15 14 38" src="https://cloud.githubusercontent.com/assets/2467194/16987474/bd89b1da-4e8c-11e6-8c97-7f123458e34a.png">